builder/app/javascript/packs/application.js

```diff
@@ -7,18 +7,13 @@
 // To reference this file, add <%= javascript_pack_tag 'application' %> to the appropriate
 // layout file, like app/views/layouts/application.html.erb
 
-import Vue from 'vue/dist/vue.esm'
-import Vuex from 'vuex'
-import axios from 'axios'
-import matestackEventHub from 'js/event-hub'
-import componentMixin from 'component/component'
+import 'core-js';
```

Comment:
Is 'core-js' needed/used in the babel-process @sigfriedCub1990?

Reply:
Here we need to import `core-js` in order to import polyfills to older browsers. So, yes, it's needed. :)